coffeecard/CoffeeCard.Library/Services/v2/AccountService.cs

```diff
@@ -275,5 +280,37 @@ private static string EscapeName(string name)
         {
             return name.Trim('<', '>', '{', '}');
         }
+
+        public async Task UpdatePriviligedUserGroups(WebhookUpdateUserGroupRequest request)
+        {
+            if (!_memoryCache.TryGetValue(AccountsWebhooksHashCacheKey, out string cachedHash))
+            {
+                var cacheExpiryOptions = new MemoryCacheEntryOptions
+                {
+                    AbsoluteExpiration = DateTime.UtcNow.AddHours(48),
+                    SlidingExpiration = TimeSpan.FromHours(24)
+                };
+
+                Log.Information("Set {AccountsWebhooksHashCacheKey} in Cache", AccountsWebhooksHashCacheKey);
+                _memoryCache.Set(AccountsWebhooksHashCacheKey, request.Hash, cacheExpiryOptions);
+            }
+
+            if (request.Hash.Equals(cachedHash))
+            {
+                Log.Information($"Hash seen before: {request.Hash}");
```

Comment:
I don't think we need this log statement. If you think we do, then maybe find a more descriptive text

Reply:
I have updated this. In fact, I had it locally but forgot to commit it at midnight 😢 

I want to keep some logging for duplicate requests as I think it can be useful in cases where the board approaches us with the question "why is X happening?" then we have at least some debugging information without having access to their implementation 😅